spec/classes/katello_application_spec.rb

```diff
@@ -104,6 +105,33 @@
           end
         end
 
+        context 'with rest client timeout' do
+          let :params do
+            base_params.merge(:rest_client_timeout => 3600)
```

Comment:
A better test would be to change the value to something else so you know it's not accidentally using a hardcoded value.